java/src/jmri/jmrit/symbolicprog/tabbedframe/PaneProgFrame.java

```diff
@@ -148,12 +148,15 @@ public boolean doPrompt() {
         resetMenu.setEnabled(false);
 
         // Add a save item
-        fileMenu.add(new AbstractAction(Bundle.getMessage("MenuSave")) {
-            @Override
-            public void actionPerformed(ActionEvent e) {
+        JMenuItem menuItem = new JMenuItem(Bundle.getMessage("MenuSaveNoDots"));
+        menuItem.addActionListener(e -> {
                 storeFile();
+                System.err.println("STORED");
```

Comment:
Static analysis fails on this line.

Reply:
Thanks. Left some debug output in. Removed.